.github/workflows/_get-changed-files.yml

```diff
@@ -26,17 +26,26 @@ jobs:
             # Get the PR number from the github context
             PR_NUMBER="${{ github.event.number }}"
 
-            # Use gh CLI to get changed files in the PR with explicit repo
-            CHANGED_FILES=$(gh api repos/${{ github.repository }}/pulls/$PR_NUMBER/files --paginate --jq '.[] | select(.status != "removed") | .filename' | tr '\n' ' ' | sed 's/ $//')
-
-            if [ -z "$CHANGED_FILES" ]; then
-              echo "No changed files found, setting to '*'"
-              CHANGED_FILES="*"
+            # Check if the PR has the lint-all-files or Reverted label
+            # Reverted PRs probably have a high likelihood of needing to have all files linted, so we should lint all files
+            HAS_LINT_ALL_LABEL=$(gh api repos/${{ github.repository }}/pulls/$PR_NUMBER --jq '.labels[] | select(.name == "lint-all-files" or .name == "Reverted") | .name')
```

Comment:
With these changes we're coupling this workflow very tightly to the linter jobs.  That's fine on it's own, but consider renaming this workflow to reflect that it shouldn't be used for anything other than lint

Or a better fix could be to pass in a list of "bypass_labels", and if any of those are on the PR then you return all files. The lint workflow can then pass in "revert" and "lint-all-files" as the bypass labels